src/calculateBullAndCows.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const calculateBullAndCows = (correct, guess) => [...guess].reduce(
+  (result, nextDigit, index) => {
+    if (nextDigit === correct[index]) {
+      result.bulls++;
+
+      return result;
+    }
+
+    if (correct.includes(nextDigit)) {
+      result.cows++;
+
+      return result;
+    }
```

Comment:
```suggestion
    result.bulls += currentDigit === correct[index];
    result.cows += (
      currentDigit !== correct[index] && correct.includes(currentDigit)
    );
```

Boolean `true` converts to 1, `false` to 0.